R/zchunk_LA101.EIA_SEDS.R

```diff
@@ -34,48 +36,68 @@ module_gcam.usa_LA101.EIA_SEDS_DISABLED <- function(command, ...) {
     A_fuel_conv <- get_data(all_data, "gcam-usa/A_fuel_conv")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` and/or 'match' calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Merge-and-Match
+
+    # Define vector to be used for filtering
+    GCAM_year <- 1971:2010
+
+    # Prep for output tables - add columns for GCAM sector and fuel names, using the substrings of the MSN code, and filter out U.S.
+    EIA_use_all_Bbtu %>%
+      gather(year, value, 4:55) %>%
```

Comment:
Please don't use hard-coded column numbers like this. Do `gather(year, value, -col_name1, -col_name2, -col_name3)` or something similar.